client/src/app-components/ticket-list.js

```diff
@@ -21,7 +22,10 @@ class TicketList extends React.Component {
         type: React.PropTypes.oneOf([
             'primary',
             'secondary'
-        ])
+        ]),
+        showClosedTickets: React.PropTypes.bool,
+        filterClosedTickets: React.PropTypes.bool,
```

Comment:
is this prop necessary? Should it be enough to check if `onShowClosedTicketsChange` or `showClosedTickets` is undefined?